src/services/products/billing/types/billingResponseTypes.ts

```diff
@@ -0,0 +1,18 @@
+export interface BillingAccountResponse{
+  first_name: string,
```

Comment:
Should these be snake case or camel case here?

Reply:
Depends on the pydantic model. Our python standards dictate using snake_case